packages/core-users/src/module/configureUsersModule.ts

```diff
@@ -12,6 +13,22 @@ import { FileDirector } from 'meteor/unchained:file-upload';
 import { Context } from '@unchainedshop/types/api';
 import { UsersCollection } from '../db/UsersCollection';
 
+const USER_EVENTS = [
+  'USER_UPDATE',
+  'USER_UPDATE_PROFILE',
+  'USER_ADD_ROLES',
+  'USER_UPDATE_ROLE',
+  'USER_UPDATE_TAGS',
+  'USER_UPDATE_AVATAR',
+  'USER_UPDATE_HEARTBEAT',
+  'USER_UPDATE_BILLING_ADDRESS',
+  'USER_UPDATE_LAST_CONTACT',
```

Comment:
Looks good to me, i'd rename "cleanConfidentialData" to "removeConfindentialServiceHashes" to make it a little bit more clear what get's claned, but that's it.

Reply:
I made it general in case we wanted to remove other data, but this name is descriptive for the current purpose, coming up.